pkg/config/config.go

```diff
@@ -17,3 +17,8 @@ type CliConfig struct {
 	GitPrivateKeyFile   string
 	GitKnownHostsFile   string
 }
+
+// CreateCliConfig is the config we use for create specific configurations.
```

Comment:
```suggestion
// CreateClusterCLIConfig is the config we use for create cluster specific configurations.
```
we could also put this in the `cli` package